Remix_Documentation/Remix.md

```diff
@@ -0,0 +1,60 @@
+# Remix
+
+HyperDX comes with simple to use out-of-the box Remix support via our [browser SDK](https://www.hyperdx.io/docs/install/browser). The browser SDK allows you to intrument your frontend application to send events, route logs, and session data to HyperDX. We also have multiple backend integrations depending on your Remix Stack, for example when using [Node.js](https://www.hyperdx.io/docs/install/javascript) servers like Express, Vercel, Netlify, Architect, etc.
+
+## Getting Started
+
+### Install
+
+```bash
+npm install @hyperdx/browser
+```
+
+### Initialize HyperDX in root.tsx file
```

Comment:
Would this be more appropriate in [`entry.client.tsx`](https://remix.run/docs/en/main/file-conventions/entry.client)?